zerver/views/messages.py

```diff
@@ -355,29 +355,9 @@ def highlight_string_text_offsets(text, locs):
     result += string[pos:]
     return result
 
-def highlight_string_bytes_offsets(text, locs):
-    # type: (AnyStr, Iterable[Tuple[int, int]]) -> Text
-    string = force_bytes(text)
-    highlight_start = b'<span class="highlight">'
-    highlight_stop = b'</span>'
-    pos = 0
-    result = b''
-    for loc in locs:
-        (offset, length) = loc
-        result += string[pos:offset]
-        result += highlight_start
-        result += string[offset:offset + length]
-        result += highlight_stop
-        pos = offset + length
-    result += string[pos:]
-    return force_text(result)
-
 def highlight_string(text, locs):
     # type: (AnyStr, Iterable[Tuple[int, int]]) -> Text
-    if settings.USING_PGROONGA:
-        return highlight_string_bytes_offsets(text, locs)
-    else:
-        return highlight_string_text_offsets(text, locs)
+    return highlight_string_text_offsets(text, locs)
```

Comment:
I think we can actually remove `highlight_string_text_offsets` and just have this be `highlight_string`.  I went ahead and did that and merged this, thanks @kou!